swarm/storage/ldbstore.go

```diff
@@ -206,7 +225,7 @@ func (s *LDBStore) updateIndexAccess(index *dpaDBIndex) {
 }
 
 func getIndexKey(hash Key) []byte {
-	hashSize := len(hash)
+	hashSize := len(hash[:])
```

Comment:
Just curious, why this change? Isn't it equivalent?

Reply:
yeah its probably an artifact from some experimentation. I'll remove.